### ISSUE TITLE
Doesn't work with XCode 10

### ISSUE THREAD

**Issue by Nik99** [TheDOme6/device_id#2]
This plugin doesn't work with XCode 10 because the swift used seems to be old.
I can't compile it with XCode 10 in release mode for iPhone XS with iOS 12.0

**Comment by Nik99** [comment]
Sorry, my bad. I made a mistake.

**Comment by Nik99** [comment]
No, the problem is still there. The swift language version is not specified, and when i try to specify it it gives 'device_id/device_id-Swift.h' not found. Any fix on this ?

**Comment by princeofgiri** [comment]
@Nik99 I have the same issue with you

**Comment by matte5031** [comment]
does it work for anyone?

**Comment by BaeksunYou** [comment]
Xcode's output:
↳
    === BUILD TARGET device_id OF PROJECT Pods WITH CONFIGURATION Debug ===
    /Users/baeksunyu/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/device_id-0.1.3/ios/Classes/DeviceIdPlugin.m:2:9: fatal error: 'device_id/device_id-Swift.h' file not found
    #import <device_id/device_id-Swift.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.

**Comment by vinhld2019** [comment]
https://stackoverflow.com/questions/53648346/flutter-ios-build-fails-in-xcode-10-1-with-swift-version
Hope it can help

**Comment by SarcasticNoodle** [comment]
flutter/flutter#18673
